### ISSUE TITLE
Keyboard navigation does not work for optgroup options

### ISSUE THREAD

**Issue by tj-shapeways** [lcdsantos/jQuery-Selectric#154]
Keyboard navigation does not work when the options are organized in optgroups. [See Fiddle.](https://jsfiddle.net/dLnc4qtt/1/) (Try to navigate with up and down keys. This seems to because the `items` collection is a tree structure of optgroups and options.

It may be possible to use the `lookupItems` collection to run through all available options:

```
   if (isPrevKey) {
    goToItem = _this.utils.previousEnabledItem(_this.lookupItems, idx);
   }

  if (isNextKey) {
    goToItem = _this.utils.nextEnabledItem(_this.lookupItems, idx);
  }
```
